### PR TITLE
ElasticBeanstalk option for only_create_app_version

### PR DESCRIPTION
**Pull request by georgantasp**
Hey Gents,

These commit add an only_create_app_version option to the elastic beanstalk provider. This makes it so the version is uploaded without making it active.

Thanks.

### CODE REVIEW

File: lib/dpl/provider/elastic_beanstalk.rb [hunk 1]
```diff
@@ -27,6 +27,10 @@ def check_auth
       def check_app
       end
 
+      def only_create_app_version
+        options[:only_create_app_version] || false
```
Comment (BanzaiMan) [inline lib/dpl/provider/elastic_beanstalk.rb]: `|| false` seems unnecessary, as the value defaults to `nil` is the key doesn't exist.

**Comment by BanzaiMan** [comment]
Could you also squash these commits, and add a document on README, please?

**Comment by georgantasp** [comment]
Squashed (i think?) and confirming the README was updated as well.

Thanks.

**Comment by georgantasp** [comment]
Is there anything else I can do to have this merged?

**Pull request merged**